keyboards/bandominedoni/bandominedoni.c

```diff
@@ -0,0 +1,119 @@
+/* Copyright 2021 3araht
```

Comment:
These files look identical to the ones in the subfolder, and as such, can probably be removed.

Reply:
My bad, I deleted them.